aggregator/src/aggregator/http_handlers.rs

```diff
@@ -168,7 +168,7 @@ impl Handler for Error {
 /// Firefox has the highest Max-Age cap, at 24 hours, so we use that. Our CORS preflight handlers
 /// are tightly scoped to relevant endpoints, and our CORS settings are unlikely to change.
 /// See: <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age>.
-const CORS_PREFLIGHT_CACHE_AGE: u32 = 24 * 60 * 60;
+const CORS_PREFLIGHT_CACHE_AGE: &str = "86400"; //24 * 60 * 60;
```

Comment:
Let's delete the trailing comment and fold it into the doccomment, like
> Firefox has the highest Max-Age cap, at 34 hours, so we use that (86400 = 24 * 60 * 60).

Reply:
Fixed in 4d1ba6e03e74964e9c97d1c936aefc3dcce2d365